plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/BannedElement.java

```diff
@@ -70,5 +72,24 @@ String getMarkerId() {
   int getSeverity() {
     return severity;
   }
+  
+  @Override
+  public boolean equals(Object object) {
```

Comment:
Needs Javadoc explaining what equals is based on (markerId and line and column number)